lib/auth/oidc.go

```diff
@@ -903,6 +903,21 @@ func (a *Server) getClaims(oidcClient *oidc.Client, connector types.OIDCConnecto
 	return claims, nil
 }
 
+// Returns a Oauth2 client.  If is a Ping provider sets the client_secret_post auth method
+func (a *Server) getOAuthClient(oidcClient *oidc.Client, connector types.OIDCConnector) (*oauth2.Client, error) {
+
+	oac, err := oidcClient.OAuthClient()
```

Comment:
We'll need to check the error here, if it fails we'll end up calling `SetAuthMethod` on a nil pointer and panic.

Reply:
check added